manifest.json

```diff
@@ -8,5 +8,7 @@
     "scripts": ["dont-record-my-search.js"],
     "persistent": false
   },
-  "manifest_version": 2
```

Comment:
Why did you delete this line of code?

Reply:
The line already exists at line number 2.

Reply:
Good enough